_editorial_policies/01_bylaws.md

```diff
@@ -44,6 +44,7 @@ either Managing Editors or Associate Editors.
 1. Selection process
     1. Members of the Editorial Board will be elected by a two-thirds vote of the Editorial Board.
         1. Managing Editors and Section Lead Editors will be elected by a two-thirds vote of the entire Editorial Board by the other current members of the Editorial Board when there is a vacancy in either capacity.
+        2. Because the Journal is currently published by the University of Colorado Boulder (UCB), at least one Managing Editor shall be a faculty member at UCB, unless and until UCB assigns its interest to another party. 
```

Comment:
@mrshirts just don't leave UCB!

Reply:
I'm not seeing a likely reason for me to leave in the foreseeable future :)